cmd/guacingest/cmd/ingest.go

```diff
@@ -90,7 +90,10 @@ func ingest(cmd *cobra.Command, args []string) {
 	defer csubClient.Close()
 
 	emit := func(d *processor.Document) error {
-		return ingestor.Ingest(ctx, d, opts.graphqlEndpoint, csubClient)
+		if err := ingestor.Ingest(ctx, d, opts.graphqlEndpoint, csubClient); err != nil {
+			logger.Errorf("unable to ingest document %q : %v", d.SourceInformation.Source, err)
```

Comment:
We should combine this with the work that @nathannaveen is doing to improve the logging. Basically, there will now be a child logger (per document) that each log will fall under. In that case, we should move the log of this error to `ingestor.Ingest` as the child logger will be instantiated there via https://github.com/guacsec/guac/blob/main/pkg/ingestor/ingestor.go#L38 once his PR merges

Reply:
Based on discussion, this will be handled by passing the logger via the  `processor.Document`. So the logger here will be replaced by the child logger. cc @nathannaveen